lib/ansible/plugins/inventory/yaml.py

```diff
@@ -9,16 +9,15 @@
     version_added: "2.4"
     short_description: Uses a specific YAML file as an inventory source.
     description:
-        - "YAML based inventory, SHOULD starts with the C(all) group and has hosts/vars/children entries."
+        - "YAML-based inventory, should start with the C(all) group and contain hosts/vars/children entries."
         - Host entries can have sub-entries defined, which will be treated as variables.
         - Vars entries are normal group vars.
         - "Children are 'child groups', which can also have their own vars/hosts/children and so on."
         - File MUST have a valid extension, defined in configuration
     notes:
-        - It takes the place of the previously hardcoded YAML inventory.
-        - To function it requires being whitelisted in configuration.
-        - Beginning with the C(all) group is the only way to set vars for the
-          C(all) group inside the inventory file.
+        - Replaces the previously hardcoded YAML inventory.
+        - Must be whitelisted in configuration to function.
```

Comment:
we can remove these 2 lines, by now no one remembers it used to be hardcoded and it is whitelisted by default (also docs explain this in main page).

Reply:
cool - I removed the line about how it used to be hardcoded and changed the note about whitelisting